### DISCUSSION TITLE
Opensubsonic

### DISCUSSION THREAD

**Discussion by bartmans** [kagemomiji/airsonic-advanced#588] [General]
Hi,

What are your thoughts on Opensubsonic: https://opensubsonic.netlify.app/docs/

**Comment by uSlackr** [comment]
FWIW, the home assistant Music Assistant supports the api. I’d like it to work with AA.


\\Greg
________________________________
From: bartmans ***@***.***>
Sent: Sunday, September 8, 2024 4:25:09 AM
To: kagemomiji/airsonic-advanced ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [kagemomiji/airsonic-advanced] Opensubsonic (Discussion #588)


Hi,

What are your thoughts on Opensubsonic: https://opensubsonic.netlify.app/docs/
